app/utils.ts

```diff
@@ -45,3 +45,7 @@ export function useMatchesData(
 export function validateEmail(email: unknown): email is string {
   return typeof email === "string" && email.length > 3 && email.includes("@");
 }
+
+export function isEmptyString(val: string): boolean {
+  return val !== undefined && val.trim().length <= 0;
```

Comment:
Out of curiosity, why test if val is not undefined if val can only be a string?

Reply:
i think the real answer here is habit 😅 . i'll make this a more generic function.